seed_db.sh

```diff
@@ -0,0 +1,249 @@
+            And this was the gold from our mining: ‘Thou mayest.’ ‘Thou mayest rule over sin.’ ');"
+
+        docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "INSERT INTO story_translation_contents (story_translation_id, line_index, translation_content) VALUES ('$STORY_TRANSLATION_ID', 5, 'The old gentlemen smiled and nodded and felt the years were well spent. It brought them out of their
+            Chinese shells too, and right now they are studying Greek.” ');"
+        
+        docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "INSERT INTO story_translation_contents (story_translation_id, line_index, translation_content) VALUES ('$STORY_TRANSLATION_ID', 6, 'Samuel said, “It’s a fantastic story. And I’ve tried to follow and maybe I’ve missed somewhere. 
+            Why is this word so important?”');"
+        
+        docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "INSERT INTO story_translation_contents (story_translation_id, line_index, translation_content) VALUES ('$STORY_TRANSLATION_ID', 7, 'Lee’s hand shook as he filled the delicate cups. 
+            He drank his down in one gulp. “Don’t you see?” he cried. ');"
+        
+        docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "INSERT INTO story_translation_contents (story_translation_id, line_index, translation_content) VALUES ('$STORY_TRANSLATION_ID', 8, '“The American Standard translation orders men to triumph over sin, 
+            and you can call sin ignorance. ');"
+        
+        docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "INSERT INTO story_translation_contents (story_translation_id, line_index, translation_content) VALUES ('$STORY_TRANSLATION_ID', 9, 'The King James translation makes a promise in ‘Thou shalt,’ 
+            meaning that men will surely triumph over sin. ');"
+        
+        docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "INSERT INTO story_translation_contents (story_translation_id, line_index, translation_content) VALUES ('$STORY_TRANSLATION_ID', 10, 'But the Hebrew word, the word timshel—‘Thou mayest’—that gives a choice. 
+            It might be the most important word in the world. 
+            That says the way is open. That throws it right back on a man.
+            For if ‘Thou mayest’—it is also true that ‘Thou mayest not.’ Don’t you see?”');"
+    done 
+
+    for STORY_TRANSLATION_ID in 1 4 6 8 9 10  
+    do 
+        docker exec -it planet-read_db_1 psql -U postgres -d planet-read -c "INSERT INTO story_translation_contents (story_translation_id, line_index, translation_content) VALUES ('$STORY_TRANSLATION_ID', 1, '');"
```

Comment:
nit: you can nest a `for i in 1 2 3 4 5 6 7 8 9 10` and use `i` as the `line_index` here to reduce the repeat code.